src/components/RadioButton/__stories__/radioButton.stories.mdx

```diff
@@ -0,0 +1,122 @@
+  "Set one radio group option as default. Select the most likely or convenient option.",
+  "Ensure both label and input are clickable to select the option.",
+  "Place radio buttons  can be placed vertically or horizontaly, using 16px spacing.",
+]}/>
+
+<Tip>When there’s limited space or no default selection, consider using a select <Link href="/?path=/docs/components-dropdown--overview" size={Link.sizes.SMALL} withoutSpacing>Dropdown</Link> instead.</Tip>
+
+## Variants
+### States
+Radio buttons have 3 states: regular, selected, and disabled.
+<Canvas>
+  <Story name="States">
+    <RadioButton text="Regular" />
+    <RadioButton text="Selected" checked />
+    <RadioButton text="Disabled" disabled />
+  </Story>
+</Canvas>
+
+## Do’s and Don’ts
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        component:
+          <DialogContentContainer className="monday-style-radio-buttons_wrapper">
+            <RadioButton text="Item 1" />
+            <RadioButton text="Item 2" checked />
+          </DialogContentContainer>,
+        description:"Use radio buttons when only one item can be selected from a list."
+      },
+      negative: {
+        component:
+          <DialogContentContainer className="monday-style-radio-buttons_wrapper">
+            <RadioButton text="Item 1" checked />
+            <RadioButton text="Item 2" checked />
+          </DialogContentContainer>,
+        description:<>Don't use <Link href="/?path=/docs/components-checkbox--overview" withoutSpacing>checkboxes</Link> when only one item can be selected from a list. Use radio buttons instead.</>
+      }
+    },
+    {
+      positive: {
+        component:
+        <DialogContentContainer className="monday-style-radio-buttons_wrapper">
+            <RadioButton text="Item 1" checked />
+            <RadioButton text="Item 2" />
+            <RadioButton text="Item 3" />
+          </DialogContentContainer>,
+        description:"Mark the first item as selected, and make sure it’s the most common or popular action."
+      },
+      negative: {
+        component:     
+        <DialogContentContainer className="monday-style-radio-buttons_wrapper">
+            <RadioButton text="Item 1" />
+            <RadioButton text="Item 2" />
+            <RadioButton text="Item 3" />
+        </DialogContentContainer>,
+        description:<>Don’t leave all radio buttons unselcted. If you can’t offer a smart default, consider using the <Link href="/?path=/docs/components-dropdown--overview" withoutSpacing>dropdown component.</Link></>
+      }
+    }
+  ]}
+/>
+
+## Use cases and examples
+### Radio button in items list
+The user needs to select only one option.
+<Canvas>
+  <Story name="Radio button in items list">
+    <div className="monday-style-radio-buttons_wrapper-column">
+      <div>Inbox view options</div>
+      <RadioButton text="Inbox updates" checked />
```

Comment:
same comment

Reply:
Please add here name attribute for each radio button.
Set the same name for all the radio button elements in the group.
Also, please make sure you are using different name for each radio button group example (there are  a lot of radio button groups in this file).
For example: 
`
<RadioButton text="Item 1" name="radio-buttons-group-3" />
<RadioButton text="Item 2" name="radio-buttons-group-3"  />
<RadioButton text="Item 3" name="radio-buttons-group-3"  />
`